_pages/repositories.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+permalink: /repositories/
+title: repositories
+nav: true
+description: GitHub repo cards.
+---
+
```

Comment:
this is a great demo page. could you add a brief description to this page (maybe to each section) on how to use repo components? ie, which files should users add their repository info to, which liquid code should be added to the page to render each section, etc.